svgdigitizer/svgplot.py

```diff
@@ -336,10 +572,35 @@ def axis_label(axis):
             return labels[-1]
 
         return {
-            self.xlabel: axis_label(self.xlabel),
-            self.ylabel: axis_label(self.ylabel),
+            axis_symbol: axis_label(axis_symbol) for axis_symbol in self.axis_symbols
         }
 
+    @property
+    @cache
+    def _grouped_ref_points(self):
+        r"""
+        Return the reference points grouped by axis symbol.
```

Comment:
I had to read the code to understand what this does. Maybe you want to add an example here so it is clear what the returned dict looks like.